includes/admin/class-wc-payments-admin.php

```diff
@@ -254,25 +254,31 @@ public function register_payments_scripts() {
 		delete_transient( WC_Payments_Account::ERROR_MESSAGE_TRANSIENT );
 
 		$wcpay_settings = [
-			'connectUrl'            => WC_Payments_Account::get_connect_url(),
-			'connect'               => [
+			'connectUrl'             => WC_Payments_Account::get_connect_url(),
+			'connect'                => [
 				'country'            => WC()->countries->get_base_country(),
 				'availableCountries' => WC_Payments_Utils::supported_countries(),
 			],
-			'testMode'              => $this->wcpay_gateway->is_in_test_mode(),
+			'testMode'               => $this->wcpay_gateway->is_in_test_mode(),
 			// set this flag for use in the front-end to alter messages and notices if on-boarding has been disabled.
-			'onBoardingDisabled'    => WC_Payments_Account::is_on_boarding_disabled(),
-			'errorMessage'          => $error_message,
-			'featureFlags'          => $this->get_frontend_feature_flags(),
-			'isSubscriptionsActive' => class_exists( 'WC_Payment_Gateway_WCPay_Subscriptions_Compat' ),
+			'onBoardingDisabled'     => WC_Payments_Account::is_on_boarding_disabled(),
+			'errorMessage'           => $error_message,
+			'featureFlags'           => $this->get_frontend_feature_flags(),
+			'isSubscriptionsActive'  => class_exists( 'WC_Payment_Gateway_WCPay_Subscriptions_Compat' ),
 			// used in the settings page by the AccountFees component.
-			'zeroDecimalCurrencies' => WC_Payments_Utils::zero_decimal_currencies(),
-			'fraudServices'         => $this->account->get_fraud_services_config(),
-			'isJetpackConnected'    => $this->payments_api_client->is_server_connected(),
-			'accountStatus'         => $this->account->get_account_status_data(),
-			'accountFees'           => $this->account->get_fees(),
-			'showUpdateDetailsTask' => get_option( 'wcpay_show_update_business_details_task', 'no' ),
-			'wpcomReconnectUrl'     => $this->payments_api_client->is_server_connected() && ! $this->payments_api_client->has_server_connection_owner() ? WC_Payments_Account::get_wpcom_reconnect_url() : null,
+			'zeroDecimalCurrencies'  => WC_Payments_Utils::zero_decimal_currencies(),
+			'fraudServices'          => $this->account->get_fraud_services_config(),
+			'isJetpackConnected'     => $this->payments_api_client->is_server_connected(),
+			'accountStatus'          => $this->account->get_account_status_data(),
+			'accountFees'            => $this->account->get_fees(),
+			'showUpdateDetailsTask'  => get_option( 'wcpay_show_update_business_details_task', 'no' ),
+			'wpcomReconnectUrl'      => $this->payments_api_client->is_server_connected() && ! $this->payments_api_client->has_server_connection_owner() ? WC_Payments_Account::get_wpcom_reconnect_url() : null,
+			'additionalMethodsSetup' => [
+				'showTask'                    => $this->is_page_eligible_for_additional_methods_setup_task(),
```

Comment:
The other attributes are using `is`/`has` prefixes - what do you think about doing the same for this? (e.g.: `isTaskVisible`)

Reply:
Yes! 💯